projects/stef.md

```diff
@@ -0,0 +1,332 @@
+protocol for OpenTelemetry data.
+
+On the tested datasets Otel/STEF payloads are several times more compact than OTLP, with 
+comparable serialization speed. Otel/STEF payloads are also smaller and serialization is 
+faster than Otel Arrow Phase 1.
+
+Otel/STEF targets a narrower niche than OTLP or Otel Arrow and is more efficient
+for that niche. Otel/STEF is optimized for payload size and fast serialization
+and is primarily intended to be used in the last communication leg from Otel Collector
+to observability backend where large volumes of data make network cost
+savings significant.
+
+Otel/STEF supports options that allow trading speed for size. At the highest 
+speed setting it can be faster than OTLP (_including_ conversion to/from pdata), while 
+producing significantly smaller payloads.
+
+A draft specification and a prototype implementation of Otel/STEF is attached to thi
+proposal.
+
+### Goals, objectives, and requirements
+
+The project goals are the following:
+
+- Implement missing capabilities in the [prototype implementation](#prototypes-and-draft-specs).
+- Extend benchmarking and test suite to cover more variety of datasets.
+- Turn the prototype into a production grade implementation.
+- Add support for traces and logs (the prototype implements metrics only).
+- Add Otel/STEF exporter and Otel/STEF receiver to Otel Collector.
+- Publish benchmark-justified guidelines on applicability of Otel/STEF vs OTLP vs Otel
+  Arrow.
+- Refine and stabilize Otel/STEF specification and encourage its implementation
+  inside or outside Otel.
+- Evaluate additional use-cases that can be served by Otel/STEF, for example
+  direct-export from SDKs to a local file.
+
+Project non-goals:
+
+- We do not plan to offer Otel/STEF as a general-purpose replacement for OTLP or for Otel
+  Arrow.
+
+Unlike
+[OTLP](https://github.com/open-telemetry/opentelemetry-proto/blob/main/docs/design-goals.md),
+for Otel/STEF it is not a goal to be a universal protocol useful for all
+node types. We do not plan it to be used by instrumented applications 
+and thus don't need Otel/STEF to be implemented in all language SDKs. This 
+significantly reduces the total engineering cost of this project since we
+only need the Go implementation for the Otel Collector.
+
+Otel/STEF design takes into account learnings from Otel Arrow and borrows some of the
+technics (e.g. dictionary encoding and gRPC streams).
+
+## Benchmarks
+
+A fairly full-featured, non-production grade prototype Otel/STEF implementation for 
+metric data exists in Go. It demonstrates wire-size and performance improvements over 
+OTLP and Otel Arrow.
+
+A small subset of existing benchmarks is reproduced in this section. For more detailed 
+speed and size tests and comparison to other protocols see
+[benchmarks](https://github.com/tigrannajaryan/stef/tree/main/benchmarks).
+
+The benchmarks are run on the highest compression / slowest speed option.
+
+### Wire Size
```

Comment:
It's unclear to me whether this used the otel-arrow unary or stream mode (where only dictionary portions that are unknown to the server are sent).

Reply:
Stream mode for Otel Arrow.

Reply:
Added a label to the table heading to make it clear.